contracts/minter/src/contract.rs

```diff
@@ -202,7 +202,11 @@ pub fn execute_mint_sender(
             .querier
             .query_wasm_smart(whitelist.clone(), &WhitelistQueryMsg::Config {})?;
 
-        if wl_config.is_active {
+        let is_active_res: IsActiveResponse = deps
+            .querier
+            .query_wasm_smart(whitelist.clone(), &WhitelistQueryMsg::IsActive {})?;
```

Comment:
do we really need an extra query? maybe a helper instead? We already have the wl_config above.

Reply:
This is how it was before. It felt a bit strange to have a helper / computed value inside something that usually just returns data. But the gas savings might be worth it.